workspaces/redhat-argocd/plugins/argocd/package.json

```diff
@@ -77,10 +77,10 @@
     "dist",
     "config.d.ts",
     "dist-scalprum",
-    "app-config.janus-idp.yaml"
+    "app-config.backstage-community.yaml"
   ],
   "scalprum": {
-    "name": "janus-idp.backstage-plugin-argocd",
+    "name": "backstage-community.backstage-plugin-argocd",
```

Comment:
```suggestion
    "name": "backstage-community.backstage-plugin-redhat-argocd",
```

Reply:
It seems you're not using a global find and replace to find and replace all instances of "janus-idp" :( 

Please check for ALL instances and do them consistently.